tardis/adapters/sites/htcondor.py

```diff
@@ -46,13 +46,15 @@
         return htcondor_queue
 
 
-htcondor_status_codes = {'0': ResourceStatus.Error,
-                         '1': ResourceStatus.Booting,
+# According to https://htcondor.readthedocs.io/en/latest/classad-attributes/
+# job-classad-attributes.html
+htcondor_status_codes = {'1': ResourceStatus.Booting,
                          '2': ResourceStatus.Running,
-                         '3': ResourceStatus.Stopped,
+                         '3': ResourceStatus.Running,
                          '4': ResourceStatus.Deleted,
                          '5': ResourceStatus.Error,
-                         '6': ResourceStatus.Error}
+                         '6': ResourceStatus.Running,
+                         '7': ResourceStatus.Stopped}
```

Comment:
Assuming you mean the `JobStatus` in the HTCondor documentation, what is your idea to deviate from this while explicitly mentioning the documentation. It seems a bit puzzling to me, e.g. status code `3` in TARDIS now means `Running` while it is `Removing` in HTCondor and `6` was `Error` before and now is `Running` again?

Reply:
I am still working on it. Please, stay tuned.

Reply:
I have updated the description of the pull request to be more verbose on the changes addressed in this pull request. All are related to the bug report #84.
To answer your question above. The mapping between HTCondor status codes and the internal `ResourceStatus`of TARDIS was complety off. Status code 3 means actually `Removing` instead of `Removed`. So, the actual job is still running and is currently in the process of being removed by HTCondor. It is definetly not `Stopped` as implied by the old status code 3 (`Removed`). 
Similar for the HTCondor status code 6, which was `Submission_err` and now is `Transferring Output`. New is HTCondor status code 7, which is `Suspended`and translates into internal `ResourceStatus` `Stopped`. Hope, that clearifies it a bit more?